src/renderer/vite-env.d.ts

```
@@ -43,50 +43,28 @@ interface Window {
      * @param {any[]} messages - Array of message objects with role and content
      * @returns {Promise<{text: string, error?: string}>} Response from Gemini AI or error message
      */
-    sendMessageWithHistory: (
-      messages: any[]
-    ) => Promise<{ text: string; error?: string }>;
+    sendMessageWithHistory: (messages: any[]) => Promise<{ text: string; error?: string }>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)